pom.xml

```diff
@@ -194,12 +211,18 @@
         <dependency>
             <groupId>com.github.scribejava</groupId>
             <artifactId>scribejava-apis</artifactId>
-            <version>8.3.1</version>
+            <version>8.3.3</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-core</artifactId>
-            <version>5.6.0</version>
+            <version>6.4.1</version>
```

Comment:
It worked on my machine running Java 23, but not here with Java 11. Oops!
I remember there is a way in Maven to specify the target Java version, which this pom doesn't use. I will look into that, as well as revert this version.